Makefile

```diff
@@ -2,6 +2,8 @@
 	    localtest localunittest localintegration \
 	    test unittest integration
 
+PREFIX := $(DESTDIR)/usr
```

Comment:
@rhatdan According to [Filesystem Hierarchy Standard](http://refspecs.linuxfoundation.org/FHS_2.3/fhs-2.3.pdf)
Software placed in / or /usr may be overwritten by system upgrades. For this reason, local software must
not be placed outside of /usr/local without good reason.

As a local software, I think we should stick runc binary in `/usr/local`.